src/oapi/vms.rs

```diff
@@ -419,11 +419,12 @@ impl VmSpecs {
     }
 
     fn parse_dedicated_instance_additional_prices(mut self, input: &Input) -> Option<VmSpecs> {
-        // dedicated_vm_additional is expressed as a percent
+        // dedicated_vm_additional is expressed as a per thousand (‰)
+        // The good formula from (https://docs.outscale.com/en/userguide/Getting-the-Price-of-Your-Resources.html) is Instance_price + ((Instance_price x Additional_price * 1000 )/100
```

Comment:
maybe write something like:
The doc is here: LINK but seems wonr, the good formula is: `Instance_price + ((Instance_price x Additional_price * 1000 )/100`

Reply:
Ok